win32ss/user/user32/windows/cursoricon.c

```diff
@@ -7,11 +7,375 @@
  */
 
 #include <user32.h>
+#include "libpng\png.h"
 
 WINE_DEFAULT_DEBUG_CHANNEL(cursor);
 WINE_DECLARE_DEBUG_CHANNEL(icon);
 //WINE_DECLARE_DEBUG_CHANNEL(resource);
 
+#include "pshpack1.h"
+typedef struct {
+    BYTE bWidth;
+    BYTE bHeight;
+    BYTE bColorCount;
+    BYTE bReserved;
+    WORD xHotspot;
```

Comment:
This is really a union, ICO files don't have a hotspot, they store planes and bpp here.

Reply:
@whindsaks, I did not create this code section from the ```#include "pshpack1.h"``` to ```#include "poppack1.h"```, but only relocated it from below between previous lines 472 and 493. As the filename of ```cursoricon.c``` indicates, it handles both cursors and icons. If you have a specific suggestion of how to make changes to this structure, please provide it here and I can see if we can do the update within this PR. Otherwise, maybe it can wait for another PR. Thanks lots for your comments and reviews.

Reply:
I assume the actual handling in the code is correct but a comment would be nice

```suggestion
    WORD xHotspot; /* planes in .ico */
    WORD yHotspot; /* bpp in .ico */
```